py3status/request.py

```diff
@@ -63,6 +63,7 @@ def __init__(self, url, params, data, headers, timeout, auth):
             elif isinstance(e, HTTPError):
                 self._status_code = e.code
                 self._error_message = reason
+                self._response = e
```

Comment:
From what I can see this is unused.  Additionally it should not be being used by any module due to being named as 'private'

Reply:
Removed!